addons/arsenal/functions/fnc_openBox.sqf

```diff
@@ -46,6 +53,7 @@ if (isNull _displayToUse || {!isNil QGVAR(camera)}) exitWith {
     [LLSTRING(CantOpenDisplay), false, 5, 1] call EFUNC(common,displayText);
 };
 
+GVAR(center) = _center;
```

Comment:
Why?

Reply:
Drive-by change, I just moved the definition up by several lines to group the definitions together.